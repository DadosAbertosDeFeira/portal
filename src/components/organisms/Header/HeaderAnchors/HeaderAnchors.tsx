import { Link } from "atoms/Link";

const anchors = [
  { href: "/#sobre", label: "Sobre" },
  { href: "/#voluntarios", label: "Volunt√°rios" },
  { href: "/#projetos", label: "Projetos" },
  { href: "/#colabore", label: "Colabore" },
  { href: "/#contato", label: "Contato" },
  {
    href: "https://anchor.fm/cidadesabertas/",
    label: "Podcast",
    target: "_blank",
  },
  {
    href: "https://dadosabertosdefeira.medium.com",
    label: "Blog",
    target: "_blank",
  },
];

export function HeaderAnchors() {
  return (
    <ul className="z-20 my-auto flex flex-row justify-start space-x-xl text-sm font-bold">
      {anchors.map(({ href, label, ...props }) => {
        return (
          <li key={label}>
            <Link href={href} {...props}>
              {label}
            </Link>
          </li>
        );
      })}
    </ul>
  );
}
