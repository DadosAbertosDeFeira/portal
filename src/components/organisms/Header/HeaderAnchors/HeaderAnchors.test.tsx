import { render } from "@testing-library/react";

import { HeaderAnchors } from "./HeaderAnchors";

describe("HeaderAnchors", () => {
  const makeSut = () => {
    return render(<HeaderAnchors />);
  };

  it("renders correctly", () => {
    const { asFragment } = makeSut();

    expect(asFragment).toMatchSnapshot();
  });
});
