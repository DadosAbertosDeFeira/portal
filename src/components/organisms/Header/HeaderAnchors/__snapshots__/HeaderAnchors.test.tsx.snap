// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderAnchors renders correctly 1`] = `[Function]`;
